pygLatin.py

```diff
@@ -0,0 +1,11 @@
+pyg = 'ay'
+
+original = raw_input('Enter a word:')
+
+if len(original) > 0 and original.isalpha():
+    word = original.lower()
+    first = word[0]
+    new_word = (word + first + pyg)[1:]
```

Comment:
could you do this so that you don't have to slice the resulting concatenated word?